apps/analysis/migrations/0012_initial.py

```
@@ -0,0 +1,63 @@
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='TranscriptAnalysis',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True)),
+                ('updated_at', models.DateTimeField(auto_now=True)),
+                ('name', models.CharField(max_length=255)),
+                ('description', models.TextField(blank=True)),
+                ('status', models.CharField(choices=[('pending', 'Pending'), ('processing', 'Processing'), ('completed', 'Completed'), ('failed', 'Failed')], default='pending', max_length=20)),
```

Ruff (0.8.2)

Line too long (193 > 120)

(E501)